apps/app/src/components/profile/my-feeds/FeedCard.tsx

```
@@ -0,0 +1,36 @@
+import { Card } from "./card";
+
+interface FeedCardProps {
+  feed: FeedResponse;
+}
+
+export function FeedCard({ feed }: FeedCardProps) {
+  const { contentCount, curatorCount } = useFeedStats(feed.id);
+
+  const isComplete = !!(
+    feed.config &&
+    feed.config.name &&
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)